client/protocols/guidance.py

```diff
@@ -0,0 +1,178 @@
+from client.protocols.common import models
+from client.utils import auth
+from urllib.request import urlopen
+import json
+import os
+import sys
+
+TGSERVER = "http://tg-server.app.cs61a.org/"
+LOCAL_TG = "tests/tg.ok_tg"
+OK_GUIDANCE = "tests/.ok_guidance"
+
+lambda_string_key_to_func = {
+    'none': lambda info, strMisU: None, 
+    'ki': lambda info, strMisU: info['ki'],
+    'reteach': lambda info, strMisU: info['dictMisU2ReteachMsg'].get(strMisU)
+}
+  
+# If student forces guidance messages to show, we will assign treatment group number below
+guidance_flag_dict = [1]
```

Comment:
Why is this a list instead of just the number 1? Why does the variable call it a dictionary? Also, the variable name doesn't really tell me what this is. Something like `DEFAULT_TREATMENT_GROUP_NUMBER` would be more descriptive.